### ISSUE TITLE
compute_random_roll number creation

### ISSUE THREAD

**Issue by MrToph** [Dappub/fairdicegame#9]
In `compute_random_roll`, you convert the seeds from bytes to a hex string. Then you concatenate the strings and use `std::hash` to receive a number.

Wouldn't it be easier, more performant and more secure to just `xor` the 64 lower bits of both seeds to receive your `uint64_t` random number?

https://github.com/Dappub/fairdicegame/blob/360fc5b02a48bb6de047389dfe472db7b266e9cb/fairdicegame/include/fairdicegame.hpp#L70

**Comment by onigirisan** [comment]
Good idea, hash combine like boost did may be better, I will try and test it, thank you
